addons/dockable_container/dockable_panel.gd

```diff
@@ -40,6 +40,8 @@ func _exit_tree() -> void:
 	active_tab_rearranged.disconnect(_on_tab_changed)
 	tab_selected.disconnect(_on_tab_selected)
 	tab_changed.disconnect(_on_tab_changed)
+	if is_instance_valid(get_popup()):
+		get_popup().queue_free()
```

Comment:
If we move the popup to be a child of `DockablePanel` as mentioned above, this would be unnecessary.